package.json

```diff
@@ -68,37 +68,55 @@
     "example:react:router:start": "npm -C examples/react-router run start",
     "example:react:router:start:reloadsw": "npm -C examples/react-router run start-reloadsw",
     "example:react:router:start:claims": "npm -C examples/react-router run start-claims",
-    "example:react:router:start:claims:reloadsw": "npm -C examples/react-router run start-claims-reloadsw"
+    "example:react:router:start:claims:reloadsw": "npm -C examples/react-router run start-claims-reloadsw",
+    "example:solid:router:dev": "npm -C examples/solid-router run dev",
+    "example:solid:router:build": "npm -C examples/solid-router run build",
+    "example:solid:router:start": "npm -C examples/solid-router run start",
+    "example:solid:router:start:reloadsw": "npm -C examples/solid-router run start-reloadsw",
+    "example:solid:router:start:claims": "npm -C examples/solid-router run start-claims",
+    "example:solid:router:start:claims:reloadsw": "npm -C examples/solid-router run start-claims-reloadsw",
+    "example:solid:router:build:sw": "npm -C examples/solid-router run build-sw",
+    "example:solid:router:start:sw": "npm -C examples/solid-router run start-sw",
+    "example:preact:router:dev": "npm -C examples/solid-router run dev",
+    "example:preact:router:build": "npm -C examples/preact-router run build",
+    "example:preact:router:start": "npm -C examples/preact-router run start",
+    "example:preact:router:start:reloadsw": "npm -C examples/preact-router run start-reloadsw",
+    "example:preact:router:start:claims": "npm -C examples/preact-router run start-claims",
+    "example:preact:router:start:claims:reloadsw": "npm -C examples/preact-router run start-claims-reloadsw",
+    "example:preact:router:build:sw": "npm -C examples/preact-router run build-sw",
+    "example:preact:router:start:sw": "npm -C examples/preact-router run start-sw"
```

Comment:
This is crazy, man. I think at this point, we should write a simple cli script to do it

Reply:
I'll work on it once new version released

I'll also reduce the examples to have only 1 example with the router variant for all tests on each framework: like solid and preact ones